### PR TITLE
disable scans by default

### PR DESCRIPTION
**Pull request by Aloren**

### CODE REVIEW

File: src/main/java/org/springframework/data/aerospike/query/QueryEngine.java [hunk 1]
```diff
@@ -34,6 +34,15 @@
  */
 public class QueryEngine {
 
+	public static final String SCANS_DISABLED_MESSAGE =
+			"Query without a filter will initiate a scan, but scans are disabled.";
```
Comment (vasilievip) [inline src/main/java/org/springframework/data/aerospike/query/QueryEngine.java:38]: why?

Comment (tdanylchuk) [inline src/main/java/org/springframework/data/aerospike/query/QueryEngine.java:38]: please clarify why scans are disabled and how to enable them

Comment (Aloren) [reply src/main/java/org/springframework/data/aerospike/query/QueryEngine.java:38]: message updated

Comment (Aloren) [reply src/main/java/org/springframework/data/aerospike/query/QueryEngine.java:38]: message updated

File: src/main/java/org/springframework/data/aerospike/query/QueryEngine.java [hunk 2]
```diff
@@ -34,6 +34,16 @@
  */
 public class QueryEngine {
 
+	public static final String SCANS_DISABLED_MESSAGE =
+			"Query without a filter will initiate a scan. Since scans are potentially dangerous operations, they are disabled by default in spring-data-aerospike. " +
```
Comment (vasilievip) [inline src/main/java/org/springframework/data/aerospike/query/QueryEngine.java:38]: > Since scans are potentially dangerous operations

Why?

**Pull request merged**